package.json

```diff
@@ -14,6 +14,7 @@
     "react-icons": "^4.3.1",
     "react-router-dom": "^6.2.2",
     "react-scripts": "^5.0.0",
+    "react-toastify": "^8.2.0",
```

Comment:
remove unnecessary dependencies.

Reply:
Done, Thank you for pointing out such a minute flaw. 
This shows how keenly you have reviewed my code.